components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIUtil.java

```diff
@@ -2844,14 +2848,35 @@ public static void copyResourcePermissions(String username, String sourceArtifac
      * @param artifactPath API resource path
      * @throws APIManagementException Throwing exception
      */
-    public static void setResourcePermissions(String username, String visibility, String[] roles, String artifactPath)
-            throws APIManagementException {
+    public static void setResourcePermissions(String username, String visibility, String[] roles, String
+            artifactPath, Registry registry) throws APIManagementException {
```

Comment:
Method comment should be modified according to the new method signature

Reply:
Fixed.